Classes/Section Controllers/SearchBar/SearchBarSectionController.swift

```diff
@@ -13,12 +13,17 @@ protocol SearchBarSectionControllerDelegate: class {
     func didChangeSelection(sectionController: SearchBarSectionController, query: String)
 }
 
-final class SearchBarSectionController: ListSwiftSectionController<String>, SearchBarCellDelegate {
+protocol SetSearchBarTextDelegate: class {
+    func append(query: String)
```

Comment:
Why not calling `didAppend(query:)` ? `append(query:)` (as a name) confuses me a bit.